source/val/validate_interfaces.cpp

```diff
@@ -46,6 +46,45 @@ bool is_interface_variable(const Instruction* inst, bool is_spv_1_4) {
   }
 }
 
+bool ContainsPhysicalStorageBuffer(ValidationState_t& _,
```

Comment:
Could this be rewritten to use `ValidationState_t::ContainsType()`? That function handles the traversal of type and takes functor for the actual check.

Reply:
I was trying to look for a util function like this... yes, can rewrite this using that